spongebob-cli

```diff
@@ -79,21 +57,37 @@ def Play(DirectLink):
     except subprocess.CalledProcessError:
         print(colored("\nAn error occured while tying to play the video! Make sure you have mpv or youtube-dl installed.", "red"))
     
-    except KeyboardInterrupt:
-        print(colored("\nUser interruped with the operation, aborting.", "red"))
-        sys.exit()
-
+    
+@Halo(text="Downloading episode.", spinner="shark", color="green")
 def Download(source):
     try:
-        subprocess.run(["youtube-dl", source]) 
+        subprocess.run(["youtube-dl", source],
+                stdout=subprocess.DEVNULL,
+                stderr=subprocess.STDOUT)
```

Comment:
The reason this was removed from the Download function is so user can see the download speed.

Reply:
it can be removed. For me, it just seems that very verbose download is too much. Or we can add `-v` or `--verbose` option.